QALiveDeployment/templates/check_service_status_webjob.ps1

```diff
@@ -0,0 +1,49 @@
+Param(
```

Comment:
Why is this file in the templates subfolder, whereas the DevDeployment version is in the folder above? Should these be consistent?

Also do we need this duplication, can't there be a common folder referenced?

Reply:
I have moved the status check file to the root deployment folder where it does indeed belong

Reply:
Thanks